src/components/Dashboard/Health/HealthItem.js

```diff
@@ -3,31 +3,33 @@ import PropTypes from 'prop-types';
 import { Icon } from 'patternfly-react';
 import { get } from 'lodash';
 
+import { HEALTHY, DEGRADED, ERROR, UNKNOWN } from './strings';
+
 const HealthStatus = {
   0: {
-    message: 'OCS is Healthy',
+    message: HEALTHY,
     iconname: 'check-circle',
     classname: 'ok',
   },
   1: {
-    message: 'OCS health is Degraded',
+    message: DEGRADED,
     iconname: 'exclamation-circle',
     classname: 'warning',
   },
   2: {
-    message: 'OCS health is in Error State',
+    message: ERROR,
     iconname: 'exclamation-triangle',
     classname: 'error',
   },
   3: {
-    message: 'Cannot get OCS health',
+    message: UNKNOWN,
     iconname: 'exclamation-triangle',
     classname: 'error',
   },
 };
 
-const HealthBody = ({ data }) => {
-  const status = get(data, 'healthy.data.result[0].value[1]', '3');
+export const HealthItem = ({ data }) => {
```

Comment:
HealthItem should receive status (OK/WARNING/ERROR) and message so it can be reused for all dashboards